### DISCUSSION TITLE
Great Tool

### DISCUSSION THREAD

**Discussion by TH3BAT** [janoside/btc-rpc-explorer#380] [General]
Hello -
Wanted to drop a note to say thanks!, really... just love using it and like the recent upgrades. 
I would love to donate, but just not BTC :).  
If you have other options, please let me know.
--TH3BAT

**Comment by janoside** [comment]
Thanks for this - glad you value this tool!

For anyone wishing to donate to support my work on this tool and/or the hosting for the public site, I accept donations at https://donate.bitcoinexplorer.org (BTC or lightning), via Cash App using $DanJanosik, or via PayPal using @danjanosik.

Thanks!
